pypeit/spectrographs/vlt_fors.py

```diff
@@ -337,8 +340,7 @@ def config_specific_par(self, scifile, inp_par=None):
             #par['reduce']['skysub']['bspline_spacing'] = 0.6
 
         if 'lSlit' in self.get_meta_value(scifile, 'decker') or 'LSS' in self.get_meta_value(scifile, 'decker'):
-            par['calibrations']['slitedges']['sync_predict'] = 'nearest'
-
+                par['calibrations']['slitedges']['sync_predict'] = 'nearest'
```

Comment:
I think the added indentation here is a typo.  Can we add the edit from my previous suggestion so that we don't have to read the file twice?  I.e.
```
decker = self.get_meta_value(scifile, 'decker')
if 'lSlit' in decker or 'LSS' in decker:
    par['calibrations']['slitedges']['sync_predict'] = 'nearest'
```

Reply:
Oops, that extra indent must have snuck in while I was reverting. Done.